### DISCUSSION TITLE
Option to show the release date of the video

### DISCUSSION THREAD

**Discussion by treasury** [pear-devs/pear-desktop#460] [Ideas]
It would be good to see the release date of the video next to the view count and likes in the middle of the bottom player bar.

**Comment by Araxeus** [comment]
![image](https://user-images.githubusercontent.com/78568641/139303628-3b76316f-aa2c-442f-ae47-83766e1e30e2.png)

@treasury It makes the the thing kinda too long no? what do you think?

also 23.09.2021 vs 23-09-2021 ?

2021-09-23 is the original format of the date but I think that order is weird
